modal.css

```diff
@@ -0,0 +1,309 @@
+  backdrop-filter: blur(10px);
+}
+
+@media (min-width: 375px) {
+  #modal-box {
+    background-color: rgb(255, 255, 255);
+    height: 718px;
+    width: 100%;
+    max-width: 343px;
+    top: 60px;
+    left: 16px;
+    box-shadow: 0 0 rgba(241, 90, 34, 0.8);
+    transform: scale(1);
+    padding: 16px;
+    z-index: 9999999999999999;
+    transition: 200ms ease-in-out;
+  }
+
+  #modal-box.active {
+    transform: scale(1);
+  }
+
+  #modal-wrapper.active {
+    display: flex;
+  }
+
+  #modal-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  #modal-box #close-modal {
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background: none;
+    font-size: 2.25rem;
+    font-weight: bold;
+    color: #333;
+  }
+
+  #modal-header h2 {
+    width: 259px;
+    height: 82px;
+    font-family: 'Roboto', sans-serif;
+    font-weight: bold;
+    font-size: 30px;
+    line-height: 44px;
+    color: #172b4d;
+    margin-top: 0;
+  }
+
+  .close-modal {
+    margin-top: -65px;
+    color: rgb(247, 243, 243);
+  }
+
+  .popup-lang {
+    display: flex;
+    width: 90px;
+    height: 36px;
+    gap: 10px;
+    margin-left: -40px;
+    list-style-type: none;
+    margin-top: -10px;
+    margin-bottom: 50px;
+  }
+
+  .feature-image {
+    width: 311px;
+    height: 220px;
+    margin-top: -25px;
+    left: 24px;
+  }
+
+  .side-info p {
+    width: 311px;
+    height: 220px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+    color: #344563;
+    margin-top: 0;
+  }
+
+  .popup-lang a {
+    text-decoration: none;
+    border: 1px solid #979493;
+    padding: 10px 12px;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-weight: bold;
+    font-size: 13px;
+    color: #3c3a39;
+    line-height: 20px;
+    width: 39px;
+    height: 20px;
+  }
+
+  .out-links a {
+    width: 220px;
+    margin-top: 15px;
+    left: 708px;
+    padding-bottom: 10px;
+    margin-right: 12px;
+    background-color: #f55800;
+    text-decoration: none;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 3%;
+    text-align: center;
+  }
+
+  .popup-lang a:hover {
+    background-color: #f55800;
+    color: white;
+  }
+
+  .out-links a:hover {
+    background-color: white;
+    color: #f55800;
+  }
+
+  .out-links {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    right: 50%;
+  }
+
+  .out-links img {
+    padding-left: 10px;
+    padding-top: 20px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #modal-box {
+    background-color: rgb(255, 255, 255);
+    height: 631px;
+    width: 100%;
+    max-width: 1156px;
+    top: 112px;
+    left: 142px;
+    box-shadow: 0 0 rgba(241, 90, 34, 0.8);
+    transform: scale(1);
+    padding: 30px;
+    z-index: 99;
+    transition: 200ms ease-in-out;
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) suggest, kindly make sure that your portifolio matches the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&mode=design&t=7Iv0D1t3l8cw4vkj-0) as much as possible. If you notice, the cross icon that closes the popup does not show. In addition, the popup modal should have the spacing between its borders and window in all the desktop screen sizes. Kindly ensure to fix this styling 👍.
- [ ] Finally I suggest that you verify the image path to make sure that the icons are displayed as intended. This is to ensure that your portifolio matches the original design to have a better UI ☺️ 

Your design at 900px😎 
![image](https://github.com/anyars-encarta/anyars-encarta.github.io/assets/99394374/191f4530-3f18-4373-8a69-52ec9dd260da)

Original design 😎 
![image](https://github.com/anyars-encarta/anyars-encarta.github.io/assets/99394374/ce943f05-b235-4cde-9411-f2b648c0e39c)

Reply:
The icons are showing at my end. I don't know why they are not displaying at your end. I can also confirm them in the logos folder (See-live.png and See-source.png)

![Source](https://github.com/anyars-encarta/anyars-encarta.github.io/assets/29536724/53d3da76-97f0-4964-8af5-79be3cb68618)
![Showing icons](https://github.com/anyars-encarta/anyars-encarta.github.io/assets/29536724/998215f8-22d2-49ce-9bbb-bbae490edec2)